target/linux/airoha/an7581/base-files/etc/uci-defaults/04_gemtekfan

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (C) 2025 openwrt.org
+#
+
+. /lib/functions.sh
+
+board=$(board_name)
+
+case "$board" in
+gemtek,w1700k)
+    # Enable automatic fan control
+    echo 2 > /sys/class/hwmon/hwmon3/pwm1_enable
```

Comment:
All HWMON entries have shifted by 2 since the RTL8261N driver supports the thermal sensor:
```suggestion
    echo 2 > /sys/class/hwmon/hwmon5/pwm1_enable
```